src/components/todo.js

```diff
@@ -1,27 +1,35 @@
 import { getItems, setItems } from '../utilities/localStorage';
 
-const TodoElement = (content, isActive, id) => {
+const TodoElement = (content, isActive, id, completed) => {
   // the following factory function is necessary because of linters
+  console.log(completed);
```

Comment:
- [ ] For **[JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)**, kindly ensure that you remove all the `console.log` from your code base. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. 
Remember to even remove the below ones:
![Screenshot from 2023-06-15 17-48-39](https://github.com/DjibrilM/todo-app/assets/83042742/5064d7d0-75f2-48a3-b17e-175390553e9f)